### ISSUE TITLE
Configuration page on actual stable versions broken

### ISSUE THREAD

**Issue by ndurchx** [jenkinsci/aws-codecommit-trigger-plugin#66]
### Version report

Jenkins and plugins versions report:

```
Jenkins: 2.277.1
OS: Linux - 4.15.0-136-generic
---
echarts-api:5.0.1-1
matrix-auth:2.6.5
scm-api:2.6.4
plugin-util-api:2.0.0
credentials:2.3.15
junit-attachments:1.6
workflow-multibranch:2.22
display-url-api:2.3.4
aws-codecommit-trigger:3.0.5
workflow-api:2.41
jdepend:1.3.0
ws-cleanup:0.39
okhttp-api:3.14.9
ssh-credentials:1.18.1
ace-editor:1.1
copyartifact:1.46
ssh-slaves:1.31.5
bootstrap4-api:4.6.0-2
content-replace:1.6.0
warnings-ng:8.10.1
git-parameter:0.9.13
jquery-detached:1.2.1
git:4.6.0
pipeline-input-step:2.12
workflow-scm-step:2.12
schedule-build:0.5.1
pipeline-rest-api:2.19
momentjs:1.1.1
workflow-support:3.8
pipeline-graph-analysis:1.10
awseb-deployment-plugin:0.3.21
analysis-model-api:9.8.1
jdk-tool:1.5
bouncycastle-api:2.20
role-strategy:3.1
email-ext:2.82
pipeline-model-extensions:1.8.4
cloudbees-folder:6.15
script-security:1.76
github:1.33.1
monitoring:1.86.0
mailer:1.33
greenballs:1.15.1
nvm-wrapper:0.1.7
workflow-cps-global-lib:2.18
lockable-resources:2.10
font-awesome-api:5.15.2-2
timestamper:1.11.8
workflow-aggregator:2.6
ldap:2.4
pipeline-model-definition:1.8.4
resource-disposer:0.15
pipeline-stage-step:2.5
snakeyaml-api:1.27.0
workflow-basic-steps:2.23
pipeline-aws:1.43
active-directory:2.23
handlebars:1.1.1
windows-slaves:1.7
antisamy-markup-formatter:2.1
forensics-api:1.0.0
htmlpublisher:1.25
data-tables-api:1.10.23-3
workflow-cps:2.90
pam-auth:1.6
xunit:3.0.0
plain-credentials:1.7
pipeline-stage-tags-metadata:1.8.4
jjwt-api:0.11.2-9.c8b45b8bb173
durable-task:1.35
pipeline-github-lib:1.0
token-macro:2.15
build-timeout:1.20
workflow-durable-task-step:2.38
matrix-project:1.18
aws-credentials:1.28
pipeline-model-api:1.8.4
thinBackup:1.10
publish-over-ssh:1.22
popper-api:1.16.1-2
jslint:0.8.2
aws-lambda:0.5.10
git-server:1.9
command-launcher:1.5
pipeline-stage-view:2.19
github-api:1.123
apache-httpcomponents-client-4-api:4.5.13-1.0
jquery:1.12.4-1
maven-plugin:3.10
trilead-api:1.0.13
promoted-builds:3.9
dtkit-api:3.0.0
checks-api:1.6.0
cloverphp:0.5
variant:1.4
Office-365-Connector:4.14.0
s3:0.11.6
zentimestamp:4.2
publish-over:0.22
jobConfigHistory:2.26
git-client:3.6.0
aws-java-sdk:1.11.955
credentials-binding:1.24
junit:1.49
pipeline-milestone-step:1.3.2
jquery3-api:3.5.1-3
authentication-tokens:1.4
workflow-job:2.40
workflow-step-api:2.23
multiple-scms:0.6
branch-api:2.6.3
jackson2-api:2.12.1
structs:1.22
external-monitor-job:1.7
javadoc:1.6
config-file-provider:3.7.0
pipeline-build-step:2.13
jsch:0.1.55.2
Result: [Plugin:echarts-api, Plugin:matrix-auth, Plugin:scm-api, Plugin:plugin-util-api, Plugin:credentials, Plugin:junit-attachments, Plugin:workflow-multibranch, Plugin:display-url-api, Plugin:aws-codecommit-trigger, Plugin:workflow-api, Plugin:jdepend, Plugin:ws-cleanup, Plugin:okhttp-api, Plugin:ssh-credentials, Plugin:ace-editor, Plugin:copyartifact, Plugin:ssh-slaves, Plugin:bootstrap4-api, Plugin:content-replace, Plugin:warnings-ng, Plugin:git-parameter, Plugin:jquery-detached, Plugin:git, Plugin:pipeline-input-step, Plugin:workflow-scm-step, Plugin:schedule-build, Plugin:pipeline-rest-api, Plugin:momentjs, Plugin:workflow-support, Plugin:pipeline-graph-analysis, Plugin:awseb-deployment-plugin, Plugin:analysis-model-api, Plugin:jdk-tool, Plugin:bouncycastle-api, Plugin:role-strategy, Plugin:email-ext, Plugin:pipeline-model-extensions, Plugin:cloudbees-folder, Plugin:script-security, Plugin:github, Plugin:monitoring, Plugin:mailer, Plugin:greenballs, Plugin:nvm-wrapper, Plugin:workflow-cps-global-lib, Plugin:lockable-resources, Plugin:font-awesome-api, Plugin:timestamper, Plugin:workflow-aggregator, Plugin:ldap, Plugin:pipeline-model-definition, Plugin:resource-disposer, Plugin:pipeline-stage-step, Plugin:snakeyaml-api, Plugin:workflow-basic-steps, Plugin:pipeline-aws, Plugin:active-directory, Plugin:handlebars, Plugin:windows-slaves, Plugin:antisamy-markup-formatter, Plugin:forensics-api, Plugin:htmlpublisher, Plugin:data-tables-api, Plugin:workflow-cps, Plugin:pam-auth, Plugin:xunit, Plugin:plain-credentials, Plugin:pipeline-stage-tags-metadata, Plugin:jjwt-api, Plugin:durable-task, Plugin:pipeline-github-lib, Plugin:token-macro, Plugin:build-timeout, Plugin:workflow-durable-task-step, Plugin:matrix-project, Plugin:aws-credentials, Plugin:pipeline-model-api, Plugin:thinBackup, Plugin:publish-over-ssh, Plugin:popper-api, Plugin:jslint, Plugin:aws-lambda, Plugin:git-server, Plugin:command-launcher, Plugin:pipeline-stage-view, Plugin:github-api, Plugin:apache-httpcomponents-client-4-api, Plugin:jquery, Plugin:maven-plugin, Plugin:trilead-api, Plugin:promoted-builds, Plugin:dtkit-api, Plugin:checks-api, Plugin:cloverphp, Plugin:variant, Plugin:Office-365-Connector, Plugin:s3, Plugin:zentimestamp, Plugin:publish-over, Plugin:jobConfigHistory, Plugin:git-client, Plugin:aws-java-sdk, Plugin:credentials-binding, Plugin:junit, Plugin:pipeline-milestone-step, Plugin:jquery3-api, Plugin:authentication-tokens, Plugin:workflow-job, Plugin:workflow-step-api, Plugin:multiple-scms, Plugin:branch-api, Plugin:jackson2-api, Plugin:structs, Plugin:external-monitor-job, Plugin:javadoc, Plugin:config-file-provider, Plugin:pipeline-build-step, Plugin:jsch]
```

- What Operating System are you using (both controller, and any agents involved in the problem)?

```
Ubuntu 18.04.5 LTS
```

### Reproduction steps

- open the configuration page /configure

### Results

Expected result:

Working configuration page

Actual result:

With the enabled sqs trigger plugin, there are two javascript errors and the apply and submit function on the page does not work. With the disabled plugin the page works again.

Errors:
```
TypeError: ip is undefined
    <anonymous> http://deployment.xxx.it/adjuncts/903b4d4d/lib/form/repeatable/repeatable.js:159
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:116
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:111
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:93
    apply http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:76
    start http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:71
    onload http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:133
    onload http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:133
    addLoadEvent http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:132
    createSearchBox http://deployment.xxx.it/static/903b4d4d/scripts/hudson-behavior.js:2472
    <anonymous> http://deployment.xxx.it/configure:7
behavior.js:119:31
```

```
TypeError: e is undefined
    <anonymous> http://deployment.xxx.it/static/903b4d4d/scripts/hudson-behavior.js:1188
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:116
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:111
    applySubtree http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:93
    apply http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:76
    start http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:71
    onload http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:133
    onload http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:133
    addLoadEvent http://deployment.xxx.it/static/903b4d4d/scripts/behavior.js:132
    createSearchBox http://deployment.xxx.it/static/903b4d4d/scripts/hudson-behavior.js:2472
    <anonymous> http://deployment.xxx.it/configure:7
behavior.js:119:31

```

**Comment by brent80** [comment]
I think this is related to this required plugin being out of date and no longer maintained
Multiple SCMs plugin.
If you disable the plugins then the page will work

**Comment by phuonghuynh** [comment]
The latest version should fix this already

**Comment by chewting99** [comment]
I had the same issue with @ndurchx 
jenkins version: Jenkins 2.289.3
AWS CodeCommit Trigger Plugin 3.0.11

Does anyone know how to solve?
Caused: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.NoClassDefFoundError: com/amazonaws/services/sqs/AmazonSQS
  at com.ribose.jenkins.plugin.awscodecommittrigger.credentials.StandardAwsCredentials$DescriptorImpl.<init>(StandardAwsCredentials.java:88)

**Comment by phuonghuynh** [comment]
@chewting99 what is the version of your plugin "aws-credentials"? `SQS* classes` are dependencies of "aws-credentials@1.28" as described in the pom file. I will try to provide a fix in next release

**Comment by ndurchx** [comment]
The installed version of aws-credentials here is 1.32

**Comment by ndurchx** [comment]
The issue is fixed with the version 3.0.12

**Comment by phuonghuynh** [comment]
Closed due to comment https://github.com/jenkinsci/aws-codecommit-trigger-plugin/issues/66#issuecomment-1061572428
